env_source/centos7_SOURCE

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
-unset DIB_RELEASE
+export DIB_RELEASE=7
```

Comment:
centos changed the default Release what requires to explicitly set the value here.